HW04.Rmd

```diff
@@ -1,126 +1,186 @@
+F_data <- gun_deaths %>% filter(sex=="F")
+mean(F_data$age)
+```
+
+## How many white males with at least a high school education were killed by guns in 2012?
+```{r education}
+#The answer is:
+kable(gun_deaths %>% filter(sex=="M") %>% filter(race=="White") %>% filter(year=="2012") %>% filter(education !="Less than HS")%>% count(), format="markdown")
+```
+
+## Which season of the year has the most gun deaths?
+
+Assume that:
+
+* Winter = January-March
+* Spring = April-June
+* Summer = July-September
+* Fall = October-December
+
+```{r season}
+#First, I created an additional column (variable) that specifies the season:
+gun_deaths$season[gun_deaths$month %in% c("1","2","3")] <- "winter"
+gun_deaths$season[gun_deaths$month %in% c("4","5","6")] <- "spring"
+gun_deaths$season[gun_deaths$month %in% c("7","8","9")] <- "summer"
+gun_deaths$season[gun_deaths$month %in% c("10","11","12")] <- "fall"
+
+#Then, I created a summary table counting the number of deaths per season. The season with the highest numer of deaths due tu gun violence is summer:
+kable(gun_deaths %>% count(season, sort=TRUE),format="markdown",
+      col.names=c("Season","Numer of gun deaths"))
+```
+
+# These are more open ended questions from here on out, you could look at making multiple summaries or types of plots. 
+
+### Are whites who are killed by guns more likely to die because of suicide or homicide? How does this compare to blacks and hispanics?
+
+Whites that are killed by guns are more likely to die because of suicide. In contrast, black and hispanics who are killed by guns are more likely to die because of homicide. See tables below, with the percentage of total deaths by intent for each race:
+
+```{r race_2}
+#Below, I created summary tables with the number of deaths by intent for each race. I also calculated the percentage for each intent. I realize that my strategy here is a bit convoluted, so I very much appreacite any suggestions here!
+
+# Whites
+white_intent_count <- gun_deaths %>% group_by(race, intent) %>% summarise(count=n()) %>% filter(race %in% c("White")) %>% mutate(percentage = (count / sum(count))*100) %>% arrange(desc(percentage))
+kable(white_intent_count,format="markdown")
+
+# Blacks
+black_intent_count <- gun_deaths %>% group_by(race, intent) %>% summarise(count=n()) %>% filter(race %in% c("Black")) %>% mutate(percentage = (count / sum(count))*100) %>% arrange(desc(percentage))
+kable(black_intent_count,format="markdown")
+
+# Hispanics
+hispanic_intent_count <- gun_deaths %>% group_by(race, intent) %>% summarise(count=n()) %>% filter(race %in% c("Hispanic")) %>% mutate(percentage = (count / sum(count))*100) %>% arrange(desc(percentage))
+kable(hispanic_intent_count,format="markdown")
+
+```
+
+### Are police-involved gun deaths significantly different from other gun deaths? Assess the relationship between police involvement and age, police involvement and race, and the intersection of all three variables.
```

Comment:
I found this questions a bit confusing, but I'm glad that you follow a similar strategy, so maybe I wasn't as confused as I thought.